scripts/linters/python_linter.py

```diff
@@ -352,7 +354,7 @@ def _lint_py_files(self, config_pylint, config_pycodestyle):
                 python_utils.PRINT('Linting Python files %s to %s...' % (
                     current_batch_start_index + 1, current_batch_end_index))
 
-            with linter_utils.redirect_stdout(stdout):
+            with linter_utils.redirect_stdout(sys.stdout):
```

Comment:
Not an expected change right? (Please make sure to do a self review first!)

Reply:
Done.